languages/python/exercises/concept/strings/.docs/hints.md

```diff
@@ -0,0 +1,28 @@
+# Hints
+
```

Comment:
We're removing the top level headers.

```suggestion
```

Reply:
Is there a centralized style guide for this? If not, we need to make one.